docs/cep/query-guide/functions/core/sizeOfSet-stream.md

```diff
@@ -6,29 +6,45 @@ The pol2Cart function calculating the cartesian coordinates x and y for the give
 
 ## Syntax
 
-    pol2Cart(<DOUBLE> theta, <DOUBLE> rho)
-    pol2Cart(<DOUBLE> theta, <DOUBLE> rho, <DOUBLE> z)
+```sql
+pol2Cart(<DOUBLE> theta, <DOUBLE> rho)
```

Comment:
The name of the file and the title of the file is incorrect it should be pol2Cart (Stream Function)